pkgs/applications/misc/slides/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "slides";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "maaslalani";
+    repo = "slides";
+    rev = "v${version}";
+    sha256 = "0gsjq8866rdbhgl8nmawx3iqm4ww26xlnybwl16a4mwvwcai0min";
+  };
+
+  vendorSha256 = "1wld36xi7ybl8k6m9kvszhrzr492k5yl87p5vc8vkx21bnblv1qp";
+
+  doCheck = true;
+
+  buildFlagsArray = [ "-ldflags= -s -w -X=main.Version=${version}" ];
```

Comment:
You could use `ldflags = [];`. Was added recently (https://github.com/NixOS/nixpkgs/pull/125520).